src/components/ConditionallyWrap.tsx

```diff
@@ -1,9 +1,7 @@
-import React, {
-  PropsWithChildren,
-  FunctionComponent,
-  ReactElement
-} from 'react'
+import { PropsWithChildren, FunctionComponent, ReactElement } from 'react'
 
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
@bluSCALE4 I wasn't able to figure out a nice way to remove these disables, would you take a quick look

Reply:
One problem I see is that `PropsWIthChildren<{...children: ReactElement}>` is used, which duplicates the children key twice.